{{{
--- lame-3.99.5.orig/configure	2012-02-28 15:54:37.000000000 -0300
+++ lame-3.99.5/configure	2014-04-29 11:15:10.118691341 -0300
@@ -11913,6 +11913,15 @@

 fi

+case $host_cpu in
+i686*)
+ check_xmmintrin=
+ ;;
+*)
+ check_xmmintrin="xmmintrin.h"
+ ;;
+esac
+
 for ac_header in \
 		 errno.h \
 		 fcntl.h \
@@ -11922,7 +11931,7 @@
 		 sys/soundcard.h \
 		 sys/time.h \
 		 unistd.h \
-		 xmmintrin.h \
+		 $check_xmmintrin \
 		 linux/soundcard.h
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
}}}
